File: spikectl/model/requests/sound.py
from ujsonrpc import RPCRequest


class ScratchSoundBeepForTimeRequest(RPCRequest):

    def __init__(self, idx: str, duration: int, note: int, volume: int):
        super().__init__(idx, 'scratch.sound_beep_for_time', {
            'duration': duration,
            'note': note,
            'volume': volume,
        })

File: spikedev-js/src/requests/sound.js
import { RPCRequest } from '../ujsonrpc.js'

export class ScratchSoundBeepForTimeRequest extends RPCRequest {

    constructor(duration, note, volume, idx = null){
        super('scratch.sound_beep_for_time', {
            'duration': duration,
            'note': note,
            'volume': volume,
        }, idx)
    }
}